diff --git a/rtl/upstream/bram.vhd b/rtl/upstream/bram.vhd
index 33eb30d..a1a6cd6 100644
--- a/rtl/upstream/bram.vhd
+++ b/rtl/upstream/bram.vhd
@@ -76,7 +76,7 @@ BEGIN
 		clock_enable_input_a => "BYPASS",
 		clock_enable_output_a => "BYPASS",
 		intended_device_family => "Cyclone V",
-		lpm_hint => "ENABLE_RUNTIME_MOD=YES,INSTANCE_NAME="&mem_name,
+		lpm_hint => "ENABLE_RUNTIME_MOD=NO,INSTANCE_NAME="&mem_name,
 		lpm_type => "altsyncram",
 		numwords_a => numwords,
 		operation_mode => "SINGLE_PORT",
